pandas/core/generic.py

```diff
@@ -1930,25 +1930,52 @@ def to_pickle(self, path, compression='infer',
 
     def to_clipboard(self, excel=True, sep=None, **kwargs):
         """
-        Attempt to write text representation of object to the system clipboard
+        Copy object to the system clipboard.
+
+        Write a text representation of object to the system clipboard.
         This can be pasted into Excel, for example.
 
         Parameters
         ----------
-        excel : boolean, defaults to True
-                if True, use the provided separator, writing in a csv
-                format for allowing easy pasting into excel.
-                if False, write a string representation of the object
-                to the clipboard
-        sep : optional, defaults to tab
-        other keywords are passed to to_csv
+        excel : bool
+            Default setting for this argument is True.
+            If True, use the provided separator, writing in a csv format for
```

Comment:
@jorisvandenbossche I think we indenting or '-' here to separate these cases?

Reply:
Depends on the context, in this case for True/False I think it is fine as sentences (but if there are more options certainly better to explicitly use lists)

Reply:
Don't think we have a policy. But I think

    - True : use the provided separator,...
    - False : ...

looks nice.